### ISSUE TITLE
Not gaining points (confidence, etc)

### ISSUE THREAD

**Issue by 11surfboards** [voidpet/bugs#617]
My issue is that I’m trying to gain levels in my confidence, etc. The only issue is when I battle VPS, it doesn’t go up. I’m not sure if I’m doing it wrong, or it’s a bug, but idk.

My mood was around 97 for Excitement and I fought 3 Level 12 Spites and it didn’t go up? Is there anything I’m doing wrong?

**Comment by TheTrickyNicky** [comment]
Nope! Theres nothing you're doing wrong. Theres a level cap on each stat (excitement, dopamine, etc) and if you hit that cap the you can no longer boast any of your stats.

**Comment by hotonami** [comment]
None of my vps stats go up anymore actually, not just the ones I have been training recently. Can my vps hit the stat cap without training?
